### ISSUE TITLE
Use icons instead of text for reply/collapse buttons

### ISSUE THREAD

**Issue by adtac** [adtac/commento#143]
The text looks out of place and occupies too much space.

I'm no designer, so I'll need some help here. Preferably an SVG/CSS icon so that we can add it inline.

**Comment by decimoseptimo** [comment]
Being this a discuss alternative, why not just mimic their UI? It is simplier to stick with tried and true.
If you still want to use icons, why not [comment](https://fontawesome.com/icons/comment?style=regular), & [cancel](https://fontawesome.com/icons/ban?style=solid).

**Comment by adtac** [comment]
@decimoseptimo This issue has been solved in the new version. Here's what it looks like right now:

![screenshot from 2018-06-11 18-24-34](https://user-images.githubusercontent.com/7521600/41232671-ba8618dc-6da4-11e8-855f-beca9ad3597b.png)

BTW, we've moved all development [to GitLab](https://gitlab.com/commento/commento-ce). So I'm closing this issue for two reasons: this has been implemented, and all issues are now being moved to GitLab. Cheers!
